app/assets/stylesheets/galleries.scss

```diff
@@ -126,7 +126,7 @@
 
 /* The preview icon boxes on replace icon and replace character */
 .replace-icon {
-  @extend .icons-box;
+  background-color: $bg_color_icon;
```

Comment:
Was this on purpose? Doesn't seem related to the @extend at least.

Reply:
I'd have to hunt for the icons-box definition but I suspect I just copied that from it